mvpa2/mappers/procrustean.py

```diff
@@ -208,3 +208,10 @@ def _train(self, source):
             d_r = np.linalg.norm(odatas[0] - res_r)/np.linalg.norm(odatas[0])
             debug('MAP_', "%s, residuals are forward: %g,"
                   " reverse: %g" % (repr(self), d_f, d_r))
+
+
+    def _compute_recon(self):
+        """For Procrustean mapper, inverse is transpose.
```

Comment:
if it is -- please provide a unit-test which verifies that so we have some guarantee of such a promise